cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -465,3 +465,50 @@ def test_vmware_default_placement(provisioner, prov_data, provider, setup_provid
     )
     assert 'Datacenter' == provider.mgmt.get_vm(vm_name).raw.parent.parent.name, (
         'The new vm is not placed in the Datacenter root directory!')
+
+
+@pytest.mark.rhv2
+@pytest.mark.provider([RHEVMProvider], override=True,
+                      required_fields=[['cap_and_util', 'capandu_vm']])
+@pytest.mark.meta(automates=[1726590], blockers=[BZ(1726590, forced_streams=["5.10"])])
+def test_linked_clone_default(provisioner, provisioning, provider, prov_data, vm_name):
+    """ Tests provision VM from a template with the selected "Linked Clone" option.
+    The template must have preallocated disks (at least one) for this test.
+
+    Required_fields is set to [['cap_and_util', 'capandu_vm']] because template for this VM has
```

Comment:
Isn't that a big CU-24x7 VM? Maybe we can discuss offline and get a small_template that has preallocated disk. 128MB of preallocated is better than bigger ones.